### PR TITLE
Update KFP tutorial content for version 3.3

### PR DESCRIPTION
**Pull request by ptitzler**
This PR refreshes the content of the Kubeflow Pipelines tutorial.
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: pipelines/run-pipelines-on-kubeflow-pipelines/README.md [hunk 1]
```diff
@@ -91,69 +91,70 @@ Custom components are [managed in the JupyterLab UI](https://elyra.readthedocs.i
...
+The component catalog can access component specifications that are stored in the local file system or on remote sources. In this tutorial 'local' refers to the file system where JupyterLab/Elyra is running. For example, if you've installed Elyra on your laptop, local refers to the laptop's file system. If you've installed Elyra in a container image, local refers to the container's file system.
 
-#### Add components from local sources
+#### Add components from the local file system
 
-Use locally stored component specifications if there is no (immediate) need to share the specification with other users, such as during initial development. To add component specifications to the registry that are stored locally:
+Use locally stored component specifications if there is no (immediate) need to share the specification with other users, such as during initial development.
```
Comment (akchinSTC) [inline pipelines/run-pipelines-on-kubeflow-pipelines/README.md:100]:
```suggestion
Use locally stored component specifications if there is no (immediate) need to share the specification with other users, for example, during initial development and prototyping.
```

File: pipelines/run-pipelines-on-kubeflow-pipelines/README.md [hunk 2]
```diff
@@ -91,69 +91,70 @@ Custom components are [managed in the JupyterLab UI](https://elyra.readthedocs.i
...
 
 1. Open the **Pipeline components** panel using one of the approaches mentioned above.
 
-   ![Add component registry entry](doc/images/add-component-registry-entry.png)
+   ![Add component catalog entry](doc/images/add-component-catalog-entry.png)
 
-   Note that the registry already includes a few example entries. These examples are included for illustrative purposes to help you get started but you won't use them in this tutorial.
-1. Add a new component registry entry by clicking `+`.
-   The first tutorial component you are adding to the registry counts the number of rows in a file.
+   Note that the palette might already [include a few example components](https://github.com/elyra-ai/examples/tree/master/component-catalog-connectors/kfp-example-components-connector), depending on how you installed Elyra. These examples are included for illustrative purposes to help you get started but you won't use them in this tutorial.
```
Comment (akchinSTC) [inline pipelines/run-pipelines-on-kubeflow-pipelines/README.md:106]:
```suggestion
   Note that the palette may already [include a few example components](https://github.com/elyra-ai/examples/tree/master/component-catalog-connectors/kfp-example-components-connector), depending on how you installed Elyra. These examples are included for illustrative purposes to help you get started but you won't use them in this tutorial.
```

File: pipelines/run-pipelines-on-kubeflow-pipelines/README.md [hunk 3]
```diff
@@ -91,69 +91,70 @@ Custom components are [managed in the JupyterLab UI](https://elyra.readthedocs.i
    - Clicking the `Open Pipeline Components` button in the pipeline editor toolbar.
    - Searching for `Manage pipeline components` in the [JupyterLab command palette](https://jupyterlab.readthedocs.io/en/stable/user/commands.html).
 
-> You can automate the component management tasks using the [`elyra-metadata install component-registry` CLI command](https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html#managing-custom-components-using-the-elyra-cli). 
+> You can automate the component management tasks using the [`elyra-metadata install component-catalog` CLI command](https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html#managing-custom-components-using-the-elyra-cli).
```
Comment (kiersten-stokes) [inline pipelines/run-pipelines-on-kubeflow-pipelines/README.md:94]:
```suggestion
> You can automate the component management tasks using the [`elyra-metadata install component-catalogs` CLI command](https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html#managing-custom-components-using-the-elyra-cli).
```

**Review by kiersten-stokes — COMMENTED** [review]
I think https://github.com/elyra-ai/elyra/pull/2350 will affect the look of some of these screenshots (and optionally some of the wording as well) with the selection of input value versus path

**Comment by ptitzler** [comment]
> I think [elyra-ai/elyra#2350](https://github.com/elyra-ai/elyra/pull/2350) will affect the look of some of these screenshots (and optionally some of the wording as well) with the selection of input value versus path

Agreed. We'll deal with that after 3.4  was released!

**Review by kiersten-stokes — APPROVED** [review]
LGTM!

**Review by akchinSTC — APPROVED** [review]

**Pull request merged**